webapp/package.json

```diff
@@ -16,66 +16,66 @@
+    "@babel/plugin-proposal-class-properties": "7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "7.9.0",
+    "@babel/plugin-syntax-dynamic-import": "7.8.3",
+    "@babel/polyfill": "7.8.7",
+    "@babel/preset-env": "7.9.0",
+    "@babel/preset-react": "7.9.4",
+    "@babel/preset-typescript": "7.9.0",
+    "@babel/runtime": "7.9.2",
+    "@types/enzyme": "3.10.5",
+    "@types/jest": "25.1.4",
+    "@types/node": "13.9.8",
+    "@types/react": "16.9.31",
+    "@types/react-dom": "16.9.6",
     "@types/react-intl": "3.0.0",
-    "@types/react-redux": "^7.1.2",
-    "@types/react-router-dom": "4.3.4",
-    "@types/react-select": "3.0.4",
-    "@types/react-transition-group": "4.2.2",
-    "@typescript-eslint/eslint-plugin": "2.1.0",
-    "@typescript-eslint/parser": "1.13.0",
-    "babel-eslint": "10.0.2",
-    "babel-jest": "24.8.0",
-    "babel-loader": "8.0.6",
-    "babel-plugin-typescript-to-proptypes": "0.17.1",
+    "@types/react-redux": "7.1.7",
+    "@types/react-router-dom": "5.1.3",
+    "@types/react-select": "3.0.11",
+    "@types/react-transition-group": "4.2.4",
+    "@typescript-eslint/eslint-plugin": "2.26.0",
+    "@typescript-eslint/parser": "2.26.0",
+    "babel-eslint": "10.1.0",
+    "babel-jest": "25.2.4",
+    "babel-loader": "8.1.0",
+    "babel-plugin-typescript-to-proptypes": "1.3.2",
     "compass-mixins": "0.12.10",
-    "css-loader": "3.1.0",
-    "enzyme": "3.10.0",
-    "enzyme-adapter-react-16": "1.14.0",
-    "enzyme-to-json": "3.3.5",
-    "eslint": "6.0.1",
-    "eslint-import-resolver-webpack": "0.11.1",
-    "eslint-plugin-import": "2.18.0",
-    "eslint-plugin-react": "7.14.2",
+    "css-loader": "3.4.2",
+    "enzyme": "3.11.0",
+    "enzyme-adapter-react-16": "1.15.2",
+    "enzyme-to-json": "3.4.4",
+    "eslint": "6.8.0",
+    "eslint-import-resolver-webpack": "0.12.1",
+    "eslint-plugin-import": "2.20.2",
+    "eslint-plugin-react": "7.19.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "24.8.0",
-    "jest-canvas-mock": "2.1.0",
-    "jest-junit": "6.4.0",
-    "node-sass": "4.12.0",
+    "jest": "25.2.4",
+    "jest-canvas-mock": "2.2.0",
+    "jest-junit": "10.0.0",
+    "node-sass": "4.13.1",
     "react-js-to-ts": "1.4.0",
-    "sass-loader": "7.1.0",
-    "style-loader": "0.23.1",
-    "webpack": "4.35.0",
-    "webpack-cli": "3.3.5"
+    "sass-loader": "8.0.2",
+    "style-loader": "1.1.3",
+    "webpack": "4.42.1",
+    "webpack-cli": "3.3.11"
   },
   "dependencies": {
-    "core-js": "3.1.4",
+    "core-js": "3.6.4",
     "debounce-promise": "3.1.2",
     "mattermost-redux": "github:mattermost/mattermost-redux#40e5bd5a18daa919c62a06b360aa27f2e6ed3e13",
     "mattermost-webapp": "github:mattermost/mattermost-webapp#23f5f93d9f12a7e2b5623e5cee6814366abd9a0f",
-    "react": "16.8.6",
-    "react-bootstrap": "0.32.4",
-    "react-dom": "16.8.6",
-    "react-redux": "5.1.1",
-    "react-select": "3.0.4",
+    "react": "16.13.1",
+    "react-bootstrap": "1.0.0",
```

Comment:
Seems a bit weird, the number of dependencies that were updated after the last command you ran. Especially this one and a few others that got the major version bumped. @jfrerich Can you do a quick sanity test and see if the bundle correctly loads in the browser and can load one of the modals?

Reply:
@mickmister, I just did a `make deploy` from this branch and was able to create a Jira ticket from the `Create Issue Modal`.  It seems to work fine.  Aren't the dependency adjustments just a consequence of not doing an `npm-check` update in a long time?

Reply:
Well, although I was able to make deploy on my dev box, the build is now broken.  I'm very tempted to start a new PR with the latest master and dump this one..